sdb/commands/linux/per_cpu.py

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+import argparse
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class LxPerCpuPtr(sdb.SingleInputCommand):
```

Comment:
can we declare the input_type (for both commands)?

Reply:
I added the input type for the 2nd command as it expects a specific struct.
`LxPerCpuPtr` is different as it literally accepts ANY pointer type. If we wanted to be EXTRA in sdb (the Linux Kernel I think doesn't actually break that way) is to ensure that pointer type passed also has the `__percpu` type specifier in its type info. Do you think that'd be useful?

Reply:
After discussing this with @jgallag88 , we decided to check for this and print out a warning if the value is not `__percpu`.

Reply:
@jgallag88 I should have checked the drgn documentation first and the kernel but apparently this is not exactly a qualifier so we can't figure this out from drgn. As a result I won't be able to make any extra tests.